run_test.py

```diff
@@ -304,11 +304,11 @@ def main():
 
         conf = {
             'base': 'ubuntu18_py36',
-            'cuda': 'cuda113',
-            'cudnn': 'cudnn82-cuda113',
-            'nccl': 'nccl2.9-cuda113',
+            'cuda': 'cuda114',
```

Comment:
Any preference in configuration to test CUDA 11.4?

Reply:
The configuration LGTM 👍🏼